### ISSUE TITLE
Pointer Events Working Group  Charter 2021

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#249]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://w3c.github.io/charter-drafts/pe-2020.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 
 - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2018%2F12%2Fpointerevents-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fpe-2020.html), and any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security.  Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).

Communities suggested for outreach: none

Known or potential areas of concern: none

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
  https://github.com/w3c/charter-drafts

Anything else we should think about as we review? 
  Nope

**Comment by plehegar** [comment]
cc @patrickhlauke

**Comment by samuelweiler** [comment]
Privacy and security reviews completed.

As I say in https://github.com/w3c/charter-drafts/issues/317, the Devices and Sensors WG has some good experience mitigating the sorts of fingerprinting risks described in Section 13 of the pointer events spec.  I wonder if this WG should be encouraged to work with them (and add a formal liaison line item) or perhaps even merge with them outright.  Flagging this item for resolution.

I'm also disturbed to see that the only chair of the WG is one of the editors of its only spec.  

Several editorial issues filed re: the charter itself:
https://github.com/w3c/charter-drafts/issues/311
https://github.com/w3c/charter-drafts/issues/312
https://github.com/w3c/charter-drafts/issues/315
https://github.com/w3c/charter-drafts/issues/316

**Comment by himorin** [comment]
The pointed proposed charter draft has invalid link in history section for previous charter. better to be replaced with https://www.w3.org/2018/12/pointerevents-charter.html

**Comment by michael-n-cooper** [comment]
Note the IndieUI WG is closed, so should be removed from the out-of-scope statement.

Otherwise no comments from APA, over to @brewerj to complete accessibility horizontal review.

**Comment by himorin** [comment]
no comment/request from i18n.

**Comment by plehegar** [comment]
All issues have been addressed.

The Group is very very small, since it's mainly doing maintenance, so I'm grateful that @patrickhlauke is willing to help, both on the Chairs and Editors side. We rely a lot on the feedback from implementers to make changes in the spec (@patrickhlauke isn't an implementer), so not much room to mess things around.

**Comment by plehegar** [comment]
(this should remain open since the charter hasn't been approved yet)

**Comment by dontcallmedom** [comment]
[rechartered](https://lists.w3.org/Archives/Public/public-new-work/2021Mar/0010.html)
